modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -153,13 +251,35 @@ protected function register_controls() {
 		$this->end_controls_section();
 	}
 
+	private function is_active_icon_exist( $settings ) :bool {
+		return array_key_exists( 'accordion_item_title_icon_active', $settings ) && ! empty( $settings['accordion_item_title_icon_active'] ) && ! empty( $settings['accordion_item_title_icon_active']['value'] );
+	}
+
+	private function render_accordion_icons( $settings ) {
+		$icon_html = Icons_Manager::try_get_icon_html( $settings['accordion_item_title_icon'], [ 'aria-hidden' => 'true' ] );
+		$icon_active_html = $this->is_active_icon_exist( $settings )
+			? Icons_Manager::try_get_icon_html( $settings['accordion_item_title_icon_active'], [ 'aria-hidden' => 'true' ] )
+			: $icon_html;
+
+		ob_start();
+		?>
+		<span class='e-n-accordion-item-title-icon'>
+			<span class='e-n-accordion-item-title-icon-opened' ><?php echo esc_html( $icon_active_html ); ?></span>
```

Comment:
I might prefer a shorter class name like .e-open and .e-closed.
These class names get very long.